Re-add value for Ingress API Version

twz123 opened the pull request:
<!--
Thank you for contributing to helm/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->

#### What this PR does / why we need it:
It was removed in #11960, but since Kubernetes 1.14, Ingresses live under networking.k8s.io.

#### Which issue this PR fixes

#### Special notes for your reviewer:

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

maorfr commented on stable/jenkins/values.yaml:
```diff
@@ -254,6 +254,8 @@ Master:
   ingress:
     enabled: false
+    # For Kubernetes v1.14+, use 'networking.k8s.io/v1'
+    ApiVersion: extensions/v1beta1
```
please change to `apiVersion` ;)

twz123 replied:
Tried to stay in sync with the NetworkPolicy stuff... okay to diverge from that?

maorfr replied:
we are trying to go for a [best practice](https://helm.sh/docs/chart_best_practices/) chart

twz123 replied:
Okie dokie!

twz123 committed:
Re-add value for Ingress API Version

It was removed in #11960, but since Kubernetes 1.14, Ingresses live under networking.k8s.io.

Signed-off-by: Tom Wieczorek <t.wieczorek@yieldlab.de>
